.github/workflows/actions.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           ${{ runner.os }}-pip-
     - name: Install dependencies
       run: |
-        pip install tensorflow>=2.13.0
+        pip install tensorflow==2.14
```

Comment:
Do we want to keep this a `2.13` for now? Also nit, but I think `~=2.13` is slightly better practice as it will pull the latest 2.13 release always.

Reply:
NLP was using 2.14 and I wanted to keep the two consistent.